package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -361,6 +360,19 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
           this.widgetBuilder = WidgetBuilder;
         },
 
+        getParamDefn: function() {
+          if (!this.paramDefn) {
+            throw 'paramDefn is required. Call setParameterDefinition';
```

Comment:
1. 'setParameterDefinition' - is not exist. Do you mean PromptPanel#setParamDefn?
2. Need add js doc to 'getParamDefn' and 'setParamDefn' functions